bip-0009/assignments.mediawiki

```diff
@@ -32,6 +32,6 @@ State can be defined, active, failed. Dates are in UTC.
 | -
 | 2016-05-01 00:00:00
 | 2017-05-01 00:00:00
-| defined
+| active since #834624
```

Comment:
Checked that this block # is correct for segwit (but with same nit as above - depending on how we want to count this "since", inclusive or not).